src/lib/helpers.js

```diff
@@ -2,25 +2,28 @@
  * Helpers for caching and mapping vocabulary.
  */
 
-export function createHelpers({ cache, webAPI }) {
+export function createHelpers({ cache, logger, webAPI }) {
   return {
     async findDatapoint(query, last) {
-      const resp = await webAPI.get('/datapoints', {
-        params: Object.assign({}, query, {
-          $limit: 1,
-          $sort: {
-            time: last ? -1 : 1
-          }
-        })
-      })
       try {
+        const resp = await webAPI.get('/datapoints', {
+          params: Object.assign({}, query, {
+            $limit: 1,
+            $sort: {
+              time: last ? -1 : 1
+            }
+          })
+        })
         return (
           resp.data &&
           resp.data.data &&
           resp.data.data.length &&
           resp.data.data[0]
         )
-      } catch (err) {}
+      } catch (err) {
+        logger.error(err)
+        return false
```

Comment:
Remove this. Just let it fall through. Function can return undefined or a value -- rather than three possible values (false, undefined and a value).